lib/fluent/plugin/buf_file.rb

```diff
@@ -78,23 +80,23 @@ def configure(conf)
 
         if specified_directory_exists || unexisting_path_for_directory # directory
           if using_plugin_root_dir || !multi_workers_configured
-            @path = File.join(@path, 'buffer.*.log')
```

Comment:
Probably it's better creating this string object at #configure since `@path_suffix` doesn't change.